src/mui/view/TabbedView.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import compose from 'recompose/compose';
+import { Tabs, Tab } from 'material-ui/Tabs';
+import getDefaultValues from '../form/getDefaultValues';
+import translate from '../../i18n/translate';
+
+export class TabbedView extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: 0,
+        };
+    }
+
+    handleChange = (value) => {
+        this.setState({ value });
+    };
+
+    render() {
+        const { children, contentContainerStyle, record, resource, basePath, translate } = this.props;
+        return (
+            <div style={{ padding: '0 1em 1em 1em' }}>
```

Comment:
But... I copied it exactly from TabbedForm... I only deleted stuff : 

*src/mui/form/TabbedForm.js *
  ```
  render() {
        const { children, contentContainerStyle, handleSubmit, invalid, record, resource, basePath, translate } = this.props;
        return (
            <form onSubmit={handleSubmit}>
                <div style={{ padding: '0 1em 1em 1em' }}>
                    <Tabs value={this.state.value} onChange={this.handleChange} contentContainerStyle={contentContainerStyle}>
                        {React.Children.map(children, (tab, index) =>
                            <Tab key={tab.props.value} label={translate(tab.props.label)} value={index} icon={tab.props.icon}>
                                {React.cloneElement(tab, { resource, record, basePath })}
                            </Tab>
                        )}
                    </Tabs>
                </div>
                <Toolbar invalid={invalid} />
            </form>
        );
    }

```